src/proctored-exam-settings/proctoredExamSettings.scss

```diff
@@ -3,3 +3,10 @@ legend {
     font-size: 1.0rem;
     font-weight: 700;
 }
+
+.form-check:focus-within {
```

Comment:
I also wonder if, maybe, something has gone wrong in the styling in adding these React-Bootstrap components to Paragon. In the React Bootstrap documentation, I can focus on checkboxes and radio buttons, so I wonder if something is amiss. Is that something you have looked into? It might be worth it if we can leverage any existing styles in React-Bootstrap instead of writing custom SCSS.

Reply:
This isn't something I had looked into, but will now!

Reply:
I apologize that I didn't include this as a possibility in the original ticket. I do think this looks great, regardless. Thank you for fixing this!